### DISCUSSION TITLE
Hide <style> tag in head...

### DISCUSSION THREAD

**Discussion by rezaser74** [nuxt/nuxt#18375] [Questions] [answered]
Hi , i'm kindda newby in nuxt i created a nuxt project  with tailwind and when i opened the elements in browser console i saw a lots of classess added to head into my html.
i added the screenshot here.
can i hide these style tags from my html?
is there any way to hide them?

![styles](https://user-images.githubusercontent.com/89445101/213654855-28f32026-3bfd-4218-bc3f-78f85f008965.png)

**Comment by TheAlexLichter** [comment, marked as answer]
This is totally normal on dev. After building for production, there should be way less included 😋

**Reply by rezaser74** [reply]
i've just worried about performance of CEO 
thanks a lot @manniL
